CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Changed
 
+- [#19](https://github.com/increments/qiita_marker/pull/19): Allow all nodes to included in custom blocks.
```

Comment:
Don't you upgrade the version?

Reply:
@kyntk 
How about releasing it later, as we plan to make some other changes?